.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: testcontainers-python
+on:
+  - push
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    runs-on: ubuntu-16.04
```

Comment:
Let's update to a more recent version of ubuntu. 16.04 is end of life in a few months.

Reply:
done